config/datatypes_conf.xml.sample

```diff
@@ -271,6 +271,104 @@
+    <datatype extension="pepstats" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="polydot" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="preg" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="prettyseq" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="primersearch" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="showfeat" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="showorf" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="sixpack" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="strider" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="supermatcher" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="syco" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="textsearch" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="vectorstrip" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="wobble" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="wordcount" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <!-- Report formats http://emboss.sourceforge.net/docs/themes/ReportFormats.html -->
+    <datatype extension="dbmotif" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="diffseq" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="excel" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="feattable" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="motif" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="regions" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="seqtable" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="simple" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="table" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="tagseq" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <!-- Sequence formats http://emboss.sourceforge.net/docs/themes/SequenceFormats.html -->
+    <datatype extension="acedb" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="clustal" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="codata" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="embl" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="fitch" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="gcg" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="genbank" type="galaxy.datatypes.data:Text" subclass="True"/>
```

Comment:
I think we can add this as needed later and hopefully with some help from biopython or a general sniffing library. Branching from galaxy in it's own self-standing project.